server/mediaserver.go

```diff
@@ -520,7 +520,34 @@ func (s *LivepeerServer) GetNodeStatus() *net.NodeStatus {
 		cpl := cxn.pl
 		m[string(cpl.ManifestID())] = cpl.GetHLSMasterPlaylist()
 	}
-	return &net.NodeStatus{Manifests: m}
+	res := &net.NodeStatus{
+		Manifests:             m,
+		Version:               core.LivepeerVersion,
+		OrchestratorPool:      []net.StatusOrchestratorInfo{},
+		RegisteredTranscoders: []net.RemoteTranscoderInfo{},
+		LocalTranscoding:      s.LivepeerNode.TranscoderManager == nil,
+	}
+	if s.LivepeerNode.TranscoderManager != nil {
+		res.RegisteredTranscodersNumber = s.LivepeerNode.TranscoderManager.RegisteredTranscodersCount()
+		res.RegisteredTranscoders = s.LivepeerNode.TranscoderManager.RegisteredTranscodersInfo()
+	}
+	if s.LivepeerNode.OrchestratorPool != nil {
+		orchs, err := s.LivepeerNode.OrchestratorPool.GetOrchestrators(s.LivepeerNode.OrchestratorPool.Size())
```

Comment:
This will mean a RPC call to all O in the pool which seems slow and potentially inconsistent. Do we need all this information within this endpoint? Would another endpoint for `/orchestrators` be more appropriate?

Reply:
Yes, it was me being lazy here - I actually wanted just list of urls contained inside `orchestratorPool`, but this wasn't available in interface so I've took what was available. 
What do you think about adding `GetURLs` like this:
```golang
type OrchestratorPool interface {
       GetURLs() []*url.URL
	GetOrchestrators(int) ([]*OrchestratorInfo, error)
	Size() int
}
```
?

Also, I think separate `/orchestrators` endpoint makes sense if deliberately want it to hit orchestrators and check what alive (like it's happening now), but for `/status` I just wanted to some sneak peek into nodes state.

Reply:
The suggested `GetURLs` method sounds good!

Reply:
Added `GetURLs` method